.eslintrc.json

```diff
@@ -18,6 +18,18 @@
     "@typescript-eslint/no-unused-vars": [
       "error",
       { "argsIgnorePattern": "^_", "varsIgnorePattern": "_" }
+    ],
+    "max-len": [
+      "error",
+      {
+        "code": 80,
+        "tabWidth": 2,
+        "comments": 200,
```

Comment:
200 columns for comments? that’s not ok.

Reply:
It just an example.

so, all strict to 80. including comments?

Reply:
There is some open source license that is significantly more than 80 characters.

This will modify these license to fix the error.
that's not cool.

Reply:
Yes, including comments (and licenses).

It seems you have a very wide monitor, but I program exclusively on a 12 inch laptop. So here is how it looks on my machine:

<img width="1280" alt="Screen Shot 2019-06-18 at 12 59 05 PM" src="https://user-images.githubusercontent.com/80/59703896-e734e180-91c8-11e9-83ca-5ae542601492.png">